### PR TITLE
Issue 3560

### PR DESCRIPTION
**Pull request by xiam**
Part of https://github.com/getlantern/lantern/issues/3612

### CODE REVIEW

**Comment by xiam** [comment]
Hey @myleshorton, I've added a few tracking links for social buttons and bookmarks. What do you think?

**Comment by myleshorton** [comment]
Awesome @xiam! @fffw you want to have a look? I'm a bit tied up right now.

File: src/github.com/getlantern/lantern-ui/app/partials/modals.html [hunk 1]
```diff
@@ -267,9 +267,9 @@
...
-        <li class="twitter"><a target="_blank" href="https://twitter.com/intent/tweet?text={{ 'LANTERN_MOBILE_SHARE' | translate | urlencode }}">Twitter</a>Twitter</li>
-        <li class="facebook"><a target="_blank" href="http://www.facebook.com/sharer.php?u={{ 'LANTERN_MOBILE_ANDROID' | translate }}">Facebook</a>Facebook</li>
+        <li class="weibo"><a target="_blank" ng-click="trackSocialLink('weibo')" href="http://service.weibo.com/share/share.php?title={{ 'LANTERN_MOBILE_SHARE' | translate | urlencode }}">Weibo</a>Weibo</li>
+        <li class="twitter"><a target="_blank" ng-click="trackSocialLink('twitter')" href="https://twitter.com/intent/tweet?text={{ 'LANTERN_MOBILE_SHARE' | translate | urlencode }}">Twitter</a>Twitter</li>
+        <li class="facebook"><a target="_blank" ng-click="trackSocialLink('facebook')" href="http://www.facebook.com/sharer.php?u={{ 'LANTERN_MOBILE_ANDROID' | translate }}">Facebook</a>Facebook</li>
       </ul>
       <p class="download-link">
         <b>{{ 'LANTERN_MOBILE_ANDROID_TEXT' | translate }}:</b> <a target="_blank" href="{{ 'LANTERN_MOBILE_ANDROID' | translate }}">{{ 'LANTERN_MOBILE_ANDROID' | translate }}</a>
```
Comment (fffw) [inline src/github.com/getlantern/lantern-ui/app/partials/modals.html]: @xiam Do you think it's useful to also track this download link, as well as the one beneath "Get the App" button?

**Comment by myleshorton** [comment]
I'm confused why CI for valencia is failing and why your branch has an old testpackages.txt @xiam. Did you maybe branch before getting the latest from valencia?

File: src/github.com/getlantern/lantern-ui/app/partials/modals.html (hunk 1 above)
Comment (xiam) [inline src/github.com/getlantern/lantern-ui/app/partials/modals.html]: Yes, let's add that one.

**Comment by xiam** [comment]
@myleshorton perhaps I need to update this branch, let me check.

File: src/github.com/getlantern/lantern-ui/app/partials/modals.html (hunk 1 above)
Comment (xiam) [inline src/github.com/getlantern/lantern-ui/app/partials/modals.html]: Should be working now @fffw

**Comment by xiam** [comment]
@myleshorton Looks like it's working now :)

**Comment by fffw** [comment]
Not able to show UI correctly unless https://github.com/getlantern/lantern/issues/3653 is resolved.

**Comment by fffw** [comment]
LGTM!

**Pull request merged**